astropy/units/tests/test_units.py

```diff
@@ -414,7 +413,10 @@ def test_compose_failed():
 
 
 def test_compose_fractional_powers():
-    x = (u.kg / u.s ** 3 * u.au ** 2.5 / u.yr ** 0.5 / u.sr ** 2)
+    # Warning: with a complicated unit, this test becomes very slow;
+    # e.g., x = (u.kg / u.s ** 3 * u.au ** 2.5 / u.yr ** 0.5 / u.sr ** 2)
+    # takes 3 s
```

Comment:
We should open an issue for that! I'm also going to add that as a benchmark

Reply:
Isn't it good to have a complex case though? It's only 3 seconds ;)

Reply:
Complex cases are tested elsewhere - but good idea to add it as a benchmark